javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,47 @@
+//student manager
+const class07Students = [];
+function addStudentToClass(studentName) { 
+    if(studentName==='Queen') {
+        class07Students.push(studentName);
+    }
+  else if(studentName===''){
+      console.log('Please enter a valid name');
+  }
+  
+  else if(class07Students.length>=6){
+      console.log('Cannot add more students to class 07');
+  }
+  else if(toCheckStudentExist(studentName)){
+      console.log('Student '+ studentName+' is already in the class');
+  }  
+  else{
+      class07Students.push(studentName)
+      return class07Students;
+  }
+}
+
+function toCheckStudentExist(studentName){
+    for(let i=0;i<class07Students.length;i++){
+        if(class07Students[i]===studentName)
+        return true;
+    }    
+}
+
+addStudentToClass('sofia');
+addStudentToClass('Ann');
+addStudentToClass('')
+addStudentToClass('Annie');
+addStudentToClass('Jack');
+addStudentToClass('willam');
+addStudentToClass('Jack');
+addStudentToClass('kate');
+addStudentToClass('Tomas');
+addStudentToClass('Queen');
+addStudentToClass('Bella');
+console.log(class07Students);
+
+
+function getNumberOfStudents() {
```

Comment:
I believe that your intention is to print the number of students. Also you don't  use this function anywhere in your code.
```suggestion
function printNumberOfStudents() {
```

Reply:
Changed the function to printNumberOfStudents(). In homework this function ask us to just display the number of students